files/etc/init.d/elasticsearch.systemd.erb

```diff
@@ -15,7 +15,7 @@ Group=<%= @resource[:group] %>
 ExecStartPre=-/usr/share/elasticsearch/bin/elasticsearch-systemd-pre-exec
 
 ExecStart=/usr/share/elasticsearch/bin/elasticsearch \
-                                                -p <%= @resource[:pid_dir] %>/elasticsearch-<%= @resource[:instance] %>.pid \
+                                                -p <%= @resource[:pid_dir] %>/elasticsearch-<%= @resource[:instance] %>.pid --quiet \
```

Comment:
This flag will likely need to be added conditionally based upon whether Elasticsearch is running on version 5.x or 2.x, as it seems this doesn't work on 2.x.